frontend/model/notifications/templates.js

```diff
@@ -107,49 +137,70 @@ export default ({
     const rootState = sbp('state/vuex/state')
     return {
       avatarUserID: data.memberID,
+      title: rootState[data.groupID]?.settings?.groupName || L('Member added'),
       body: L('The group has a new member. Say hi to {strong_}{name}{_strong}!', {
         name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.memberID}`,
         ...LTags('strong')
       }),
+      plaintextBody: L('The group has a new member. Say hi to {strong_}{name}{_strong}!', {
```

Comment:
This notification, and probably some others, still show the site's generic grey icon w/the first letter of the site on Android. In this case maybe we should use the user's icon? Or is the problem that it's not loaded yet because they just joined? 🤔